src/components/Button/Button.tsx

```diff
@@ -0,0 +1,418 @@
+import * as React from "react"
+import styled from "@emotion/styled"
+import { css } from "@emotion/react"
+import { pxToRem } from "../ThemeProvider/typography"
+import type { Theme, ThemeOptions } from "@mui/material/styles"
+import {
+  LinkAdapter,
+  LinkAdapterPropsOverrides,
+} from "../LinkAdapter/LinkAdapter"
+
+type ButtonVariant =
+  | "primary"
+  | "secondary"
+  | "tertiary"
+  | "text"
+  | "unstable_noBorder"
+  | "unstable_inverted"
+  | "unstable_success"
```

Comment:
These are renamed relative to MIT Learn. These variants are used very rarely and are not present in Figma. I'd like to remove them.